main.py

```diff
@@ -356,7 +358,7 @@ def get_cache_client():
     global _CACHE_CLIENT
     if _CACHE_CLIENT is None:
```

Comment:
I guess we'd probably want to check CACHE_BUCKET_NAME taking get_smtp_publisher as an example

```suggestion
    if _CACHE_CLIENT is None and CACHE_BUCKET_NAME is not None:
```

Reply:
Otherwise, I don't know if y'all have discussed about this... do we want to keep using "cache_file_storage" as a default value for CACHE_BUCKET_NAME?

Reply:
Hmm we havent discussed about that, As much as I remember from my meeting with Nik.

Reply:
Also, since we're requiring the `CACHE_BUCKET_NAME` to be set, we don't need to check for it here.

Reply:
>  requiring the CACHE_BUCKET_NAME to be set

Should we be using `CACHE_BUCKET_NAME = os.environ["KCIDB_CACHE_BUCKET_NAME"]` then?

Reply:
Yes